blocks/procedures.js

```diff
@@ -934,9 +934,9 @@ Blockly.Blocks['procedures_callnoreturn'] = {
         Blockly.Events.setGroup(event.group);
         if (event.newValue) {
           this.previousDisabledState_ = this.disabled;
```

Comment:
Should update previousDisabledState_ to previousEnabledState_ in this file (with the appropriate true/false swaps)